console.log("test-1")
try{
   // let a=10;
console.log(a)
}catch(err){
    console.log("err is ",err)
}finally{
    //cleanup code
}
console.log("test-2")

console.log("test-3")
console.log("test-4")


function sample(){
    return "hello"
}

console.log(sample())